### DISCUSSION TITLE
Make the code editor pop out into a window

### DISCUSSION THREAD

**Discussion by lukewilson2002** [godotengine/godot-proposals#4501] [Editor]
I find myself editing a scene and going back to the code editor a lot, and sometimes by mistake. This proposal isn't really sophisticated, I'm just saying it would be a good QOL improvement to make the code editor pop out, so that it would be possible to Alt+Tab between your scene and your code on one monitor, or on two monitors have the editor on your left side and the scene on your right for example.

Using VS Code or another editor is an alternative, but if we have the capability, why not do it?

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/28.
